projects/laji-ui/src/lib/progressbar/progressbar/progressbar.component.ts

```diff
@@ -0,0 +1,20 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'lu-progressbar',
+  template: `
+    <div class="lu-progressbar-fill" [style.width]="Math.floor(value / max * 100) + '%'">
```

Comment:
[Could we start using the progress html element? ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress)

Reply:
Hmm... I couldn't manage to customize the styling properly. The animation doesn't work. Also it doesn't display the text - even though that mozilla doc puts a text inside it and it also doesn't show it. I don't think the default styling is that bad, but we'd lose at least the text content... Idk. I added `role="progressbar"` as a consolation